ChangeLog

```diff
@@ -1,4 +1,12 @@
-+ Opencryptoki 3.14
++ openCryptoki 3.15
+- common: conform to PKCS 11 3.0 Baseline Provider profile
+- CCA: support key wrapping
+- EP11: support m_ReEncryptSingle
```

Comment:
C_IBM_ReencryptSingle is actually available for all tokens, and is of special interest for the secure key EP11 and CCA tokens.

Reply:
right, i changed that to also mention the name of our new vendor defined interface and the actual interface name that is exported rather than the EP11 function.